web-components/src/components/table/Table.ts

```diff
@@ -53,6 +60,23 @@ export class Table extends LitElement {
     this.headerRow = this.results.data[0];
     this.csvData = this.results.data.slice(1, this.results.data.length);
     this.requestUpdate("tabledata");
+    this.linkCellItems();
+  }
+
+  get cellItem() {
+    return this.rowCell;
+  }
+
+  linkCellItems() {
+    const element = this.rowCell;
+    element?.forEach(item => {
+      this.errors.forEach(i => {
+        if (item.innerText === i) {
+          item.classList.add("error");
+          item.insertAdjacentHTML('beforeend', '<md-icon name="icon-warning_24" color="red"></md-icon>');
```

Comment:
this is a warning icon as you can see from the name (not error). Lily confirmed that for warning we should use yellow, not red

Reply:
Updated